integrations/docker-compose.static_analyzers.yml

```diff
@@ -5,9 +5,9 @@ version: '3'
 # All additional integrations should be added following this format only.
 
 services:
-  capa:
-    image: intelowlproject/intelowl_capa:${INTELOWL_TAG_VERSION}
-    container_name: intelowl_capa
+  static_analyzers:
+    image: intelowlproject/intelowl_static_analyzers:${INTELOWL_TAG_VERSION}
```

Comment:
/reminder: need to create a repository on dockerhub for this new docker container and stop autobuild of the deprecated ones.

Reply:
@mlodic can you create a new repo on dockerhub with name `intelowl_static_analyzers`, then I'll configure the auto-builds.

Reply:
disabled auto-build on dockerhub for capa and peframe repos.

Reply:
done!

Reply:
configured. :+1: